app/javascript/mastodon/locales/zh-CN.json

```diff
@@ -5,27 +5,27 @@
   "account.edit_profile": "修改个人资料",
   "account.follow": "关注",
   "account.followers": "关注者",
-  "account.follows": "正关注",
```

Comment:
I have guessed it may be intentional, but still not accustomed to it. But now I think it seems to be better. So should we change it back?